### PR TITLE
Remove lavalink from embed

### PR DESCRIPTION
**Pull request by yuansheng1549**

### CODE REVIEW

**Comment by Bocon778** [comment]
You can still use lavalink right?

**Comment by yuansheng1549** [comment]
yes, but you have to limit the ram to 2GB

**Comment by Bocon778** [comment]
So why are you removing it then?

**Comment by yuansheng1549** [comment]
because dan removed lavalink server creation, but he forgot to remove from the embed

**Comment by Bocon778** [comment]
Oh okay, makes sense.

File: bot/discord/commands/server/create-donator.js [hunk 1]
```diff
@@ -32,22 +32,25 @@ exports.run = async(client, message, args) => {
         message.channel.send("You are at your premium server limit")
         return;
     }
-
+    
+    const helpEmbed = new Discord.MessageEmbed()
+        .setColor("RED")
+        .setDescription(`List of servers: (use DBH!server create-donator <type> <name>)\n\n*Please note that some nodes might be having trouble connecting to the bot which may lead into this process giving out an error.*\n`)
```
Comment (itzJOHv) [inline bot/discord/commands/server/create-donator.js:38]: nodes dont have issues with connecting to the bot, but the bot may fail to connect to nodes

**Pull request closed without merging**